README.md

```diff
@@ -1,44 +1,28 @@
-# The Legend of Zelda: Ocarina of Time
+# Hacker Zelda: Ocarina of Time (HackerOoT)
 
-[![Build Status][jenkins-badge]][jenkins] [![Decompilation Progress][progress-badge]][progress] [![Contributors][contributors-badge]][contributors] [![Discord Channel][discord-badge]][discord]
+This project, based on the [Zelda: Ocarina of Time decompilation project](https://github.com/zeldaret/oot/), aims to be an easy-to-use base to make Zelda: Ocarina of Time mods. **It is not producing a PC port.**
 
-[jenkins]: https://jenkins.deco.mp/job/OOT/job/master
-[jenkins-badge]: https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.deco.mp%2Fjob%2FOOT%2Fjob%2Fmaster
-
-[progress]: https://zelda64.dev/games/oot
-[progress-badge]: https://img.shields.io/endpoint?url=https://zelda64.dev/assets/csv/progress-oot-shield.json
-
-[contributors]: https://github.com/zeldaret/oot/graphs/contributors
-[contributors-badge]: https://img.shields.io/github/contributors/zeldaret/oot
-
-[discord]: https://discord.zelda64.dev
-[discord-badge]: https://img.shields.io/discord/688807550715560050?color=%237289DA&logo=discord&logoColor=%23FFFFFF
-
-```diff
-- WARNING! -
-
-This repository is a work in progress, and while it can be used to make certain changes, it's still
-constantly evolving. If you use it for modding purposes in its current state, please be aware that
-the codebase can drastically change at any time. Also note that some parts of the ROM may not be
-'shiftable' yet, so modifying them could be difficult at this point.
-```
-
-This is a WIP **decompilation** of ***The Legend of Zelda: Ocarina of Time***. The purpose of the project is to recreate a source code base for the game from scratch, using information found inside the game along with static and/or dynamic analysis. **It is not producing a PC port.** For more information you can get in touch with the team on our [Discord server](https://discord.zelda64.dev).
-
-The only build currently supported is Master Quest (Debug), but other versions are planned to be supported.
+The only build currently supported is Master Quest (Debug), other versions will eventually be supported.
 
 It builds the following ROM:
 
-* zelda_ocarina_mq_dbg.z64 `md5: f0b7f35375f9cc8ca1b2d59d78e35405`
+* zelda_ocarina_mq_dbg.z64
 
 **Note: This repository does not include any of the assets necessary to build the ROM. A prior copy of the game is required to extract the needed assets.**
 
+## HackerN64
+
+**Discord:** <https://discord.gg/brETAakcXr>
+
+## Zelda Decompilation
 **Website:** <https://zelda64.dev>
 
 **Discord:** <https://discord.zelda64.dev>
 
 ## Installation
 
+Note: instructions for setting up Visual Studio Code can be found [here](docs/vscode.md).
```

Comment:
yooo nice

Reply:
You think there's any merit to linking the HackerSM64 wiki tutorial for WSL2 installation or is the OoT process too different to make sense?